### ISSUE TITLE
Unable to Install for VS 2017

### ISSUE THREAD

**Issue by GTruss** [tomasr/viasfora#225]
I don't know what happened, but I'm no longer able to install or use the extension in VS 2017. Everything was fine until a few days ago, then it just stopped working. Looking at Tools -> Options, Viasfora no longer shows up in the list. It was installed, but it's gone now.

I downloaded the most recent version of Viasfora from the VS Marketplace and reran the installer. A message comes up that says, "This extension is already installed to all applicable products." But it's not being installed for VS 2017.

I have restarted VS 2017 and rebooted the PC with no luck. I'm not familiar with extension development, so let me know if there are install log files or something I can do to help.

Visual Studio 2017, Version 15.5.2
Windows 10 Pro, Version 1703, OS Build 15063.786

**Comment by tomasr** [comment]
Any chance it is installed, but disabled? If you go into the Extensions dialog, does it show up?

Otherwise, you could try to find the folder for the extension under `C:\Users\<user>\AppData\Local\Microsoft\VisualStudio\15.0_<id>\Extensions` that belongs to Viasfora and delete the folder, and try again.

**Comment by GTruss** [comment]
Hi, Tomas. Apologies for the delayed response due to the holidays. I looked in the folder as suggested and I didn't see it; it seems as if Viasfora just didn't recognize VS 2017 as a valid target.

So, I ran through a full Repair on Visual Studio, double-clicked the VSIX file and VS 2017 was immediately recognized and the extension installed successfully. I don't know what was corrupt to cause this issue to happen, but it seems a full Repair resolves it.

Thanks for your time and effort. Will report back if this occurs again.
